hack/test-update-etcd-objects.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to test hack/update-etcd-object.sh works as expected.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+# The api version in which objects are currently stored in etcd.
+KUBE_OLD_API_VERSION=${KUBE_OLD_API_VERSION:-"v1beta1"}
+# The api version in which our etcd objects should be converted to.
+# The new api version
+KUBE_NEW_API_VERSION=${KUBE_NEW_API_VERSION:-"v1beta3"}
+
+ETCD_HOST=${ETCD_HOST:-127.0.0.1}
+ETCD_PORT=${ETCD_PORT:-4001}
+API_PORT=${API_PORT:-8080}
+API_HOST=${API_HOST:-127.0.0.1}
+KUBELET_PORT=${KUBELET_PORT:-10250}
+KUBE_API_VERSIONS=""
+RUNTIME_CONFIG=""
+
+KUBECTL="${KUBE_OUTPUT_HOSTBIN}/kubectl"
+UPDATE_ETCD_OBJECTS_SCRIPT="${KUBE_ROOT}/hack/update-etcd-objects.sh"
+
+function startApiServer() {
```

Comment:
curly brace on initial line or subsequent line-- I don't care which way it goes but it should be consistent :)